### DISCUSSION TITLE
Napalm plugin: Local_requirements.txt not present

### DISCUSSION THREAD

**Discussion by Rohrreiniger** [netbox-community/netbox#14595] [Help Wanted!] [answered]
Hi @ all!
I installed the Napalm plugin and stumbled upon this sentence "[by adding to your local_requirements.txt](https://github.com/netbox-community/netbox-napalm)". 

Am I right in assuming that if this file does not exist in /opt/netbox, where the requirements.txt is already located, that I can simply create it empty and fill it with the entry via (venv) "netbox-napalm-plugin"? TIA!

Merry Christmas

**Comment by candlerb** [comment, marked as answer]
Yes, just create `/opt/netbox/local_requirements.txt` with the names of the packages you want to install, one per line. Then run `./upgrade.sh` to rebuild your environment, which will include those packages.
